README.md

```diff
@@ -13,6 +13,21 @@ This Repository
 This repository contains the markup needed to deploy the floeproject.org website.
 
 
+Building a Docker image
+=======================
+
+You can serve the website from a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t floeproject .`
+* Run the container: `docker run --name floeproject -p 8000:80 floeproject`
+
+The website will be available at [http://localhost:8000](http://localhost:8000)
+
+If you make changes to the HTML code, repeat the steps to build the image and start a new container.
```

Comment:
The documentation is missing the information about how to remove the docker image.